### PR TITLE
added failing test for simple SWC neuron

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

**Comment by juanchopanza** [comment]
Could you mark the test as `@nottest`?

File: neurom/io/datawrapper.py [hunk 1]
```diff
@@ -72,11 +75,17 @@ def _section_end_points(data_block):
     # number of children per point
     n_children = defaultdict(int)
     for row in data_block:
-        n_children[int(row[COLS.P])] += 1
+            n_children[int(row[COLS.P])] += 1
```
Comment (juanchopanza) [inline neurom/io/datawrapper.py]: double indentation?

File: neurom/io/datawrapper.py [hunk 2]
```diff
@@ -72,11 +75,17 @@ def _section_end_points(data_block):
...
 
     # end points have either no children or more than one
-    return set(i for i, row in enumerate(data_block)
-               if n_children[row[COLS.ID]] != 1)
+    endpoints = set(i for i, row in enumerate(data_block)
+                    if n_children[row[COLS.ID]] != 1)
+
+    #manually add the 'last' point of the soma
+    last_soma_idx = np.nonzero(data_block[:, COLS.TYPE] == POINT_TYPE.SOMA)[-1]
```
Comment (juanchopanza) [inline neurom/io/datawrapper.py]: The soma could actually be split into blocks, so the "last" soma point would only catch one of the end-points.

Comment (mgeplf) [inline neurom/io/datawrapper.py]: weird

**Comment by juanchopanza** [comment]
I think this is superseded by #464.

**Pull request closed without merging**